### PR TITLE
Initial pull request

### PR DESCRIPTION
**Pull request by EmmanuelWLX**
## Summary 
- Initial kickoff
- Added tittle
- Tech component

## Screenshot
image

## Trello cards
https://trello.com/c/GQG1hLdI/11-setup-del-proyecto
https://trello.com/c/V7x6WU4B/21-page-titles
https://trello.com/c/bg7ggnQs/19-css-resets

### CODE REVIEW

File: scss/application.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@import 'equalizer';
+@import 'styles';
+@import 'titles';
+@import 'tech';
```
Comment (FrankIglesias) [inline scss/application.scss]: at tailing empty line at the end of the file

File: scss/styles.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
```
Comment (FrankIglesias) [inline scss/styles.scss]: same

File: scss/tech.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  width: 200px;
+  height: 275px;
+  text-align: center;
+  border: 1px solid $green;
+  border-radius: 6px;
+  padding: 120px 25px 80px;
+  position: relative;	
+  margin-left: 150px;
+
+  .release,
```
Comment (FrankIglesias) [inline scss/tech.scss:27]: check indentation (1 tab = 2 spaces)

File: scss/tech.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  .descripcion,
+  .nombre,
+  .tech-logo {
+  	position: relative;
+  	z-index: 2;
+  }
+
+  .release {
+  	top: 10px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: remve this

File: scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  	border-radius: 6px;
+  	position: absolute;
+  	top: 10px;
+  	left: 10px;
+  	height: 275px;
+    width: 200px;
+    z-index: 1;
+  }
+  
+}
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  	background-color: $green;
+  	border-radius: 6px;
+  	position: absolute;
+  	top: 10px;
+  	left: 10px;
+  	height: 275px;
+    width: 200px;
+    z-index: 1;
+  }
+
```
Comment (FrankIglesias) [inline scss/tech.scss]: remove empty line

File: scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  .descripcion {
+  	top: 20px;
+  	/* font-size: $text-medium; */
+  }
+
+  .tech-logo {
+    width: $tech-logo-size;
+    height: $tech-logo-size;
+    position: absolute;
```
Comment (FrankIglesias) [inline scss/tech.scss]: remove this

File: scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    top: 15px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: and this

File: scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    align-self: center;
+  }
+
+  
+  &::after {
```
Comment (FrankIglesias) [inline scss/tech.scss]: check indentation

File: scss/titles.scss [hunk 10]
```diff
@@ -0,0 +1,20 @@
+@import 'styles';
+
+.title {
+  display: flex;
+  padding: 50px 0 0 50px;
```
Comment (FrankIglesias) [inline scss/titles.scss]: this can be center aligned

File: scss/titles.scss [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+    content: '';
+    display: flex;
+    position: relative;
+    left: 20px;
+    top: -26px;
+    height: 80px;
+    width: 80px;
+    z-index: -1;
+  }
+}
```
Comment (FrankIglesias) [inline scss/titles.scss]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/tech.scss (hunk 4 above)
Comment (EmmanuelWLX) [reply scss/tech.scss]: the whole class or only the top property?

File: scss/tech.scss (hunk 7 above)
Comment (EmmanuelWLX) [reply scss/tech.scss]: the whole class? :(

File: index.html [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div>
```
Comment (FrankIglesias) [inline index.html]: this div can be deleted

File: index.html [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+    <h1 class="title">Front-End Tools</h1>
+  </div>
+  <div class="tech-box">
+    <div class="tech">
+      <img src="assets/react_icon.svg" class="tech-logo"/>
+      <h1 class="nombre">ReactJS</h1>
```
Comment (FrankIglesias) [inline index.html]: use a less important tag for tech names

File: index.html [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div>
+    <h1 class="title">Front-End Tools</h1>
+  </div>
+  <div class="tech-box">
+    <div class="tech">
+      <img src="assets/react_icon.svg" class="tech-logo"/>
```
Comment (FrankIglesias) [inline index.html]: add alt to all images

File: index.html [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+    </div>
+    <div class="tech back">
+      <img src="assets/django_icon.svg" class="tech-logo"/>
+      <h1 class="nombre">Django</h1>
+      <p class="release">Release: 2005</p>
+      <p class="descripcion">Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern.</p>
+    </div>
+  </div>
+</body>
+</html>
```
Comment (FrankIglesias) [inline index.html]: add tailing empty line

File: scss/application.scss (hunk 1 above)
Comment (FrankIglesias) [inline scss/application.scss]: add tailing empty line

File: scss/colors.scss [hunk 16]
```diff
@@ -0,0 +1,3 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
```
Comment (FrankIglesias) [inline scss/colors.scss]: add tailing empty line

File: scss/tech.scss [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+    width: 230px;
+    transition: top left 2s ease-in-out; 
+  }
+
+  &:hover::before {
+    top: 10px;
+    left: 10px;
+    transition: top left 2s ease-in-out;
+  }
+
```
Comment (FrankIglesias) [inline scss/tech.scss]: remove empty line

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Pull request closed without merging**